config/local_json.go

```diff
@@ -37,6 +37,7 @@ type ProberContextSubConfig struct {
 	Cookies map[string]string `json:"cookies"`
 	// Whether to follow http redirects from the server or not. Empty stanza uses the default 10 level redirect limit.
 	AllowRedirects bool `json:"allow_redirects,omitempty"`
+	Timeout        int  `json:"timeout,omitempty"`
```

Comment:
The default int type is signed. Can we use an unsigned type ? Using unsigned type serves as a self documenting code, which signals to the reader that timeouts can only be positive numbers.

Reply:
Actually, even better option is to use Time.ParseDuration:

https://pkg.go.dev/time#ParseDuration

Then we can specify human readable prefixes right in the config like 10s etc.

Reply:
Yep, parseDuration() is a perfect fit, added small unmarshaler for it.

Reply:
Time.ParseDuration() takes a string argument. So you can simply set the type to the base string type and then pass it into Time.ParseDuration(). What does having a separate unmarshaller achieve ?

Reply:
Sounds great, done